config/kata.yml

```diff
@@ -49,6 +46,7 @@ kata_master:
   image: "{{ kata_master_image}}"
   num_nodes: "{{ kata_master_num_nodes }}"
   user: "centos"
+  nodenet_resource: "Cluster::NodeNet3"
```

Comment:
As a future option, it would be more succinct to define a default value for the infrastructure using `cluster_params.nodenet_resource` in the parameters for `cluster-infra`.  This could be defined on an infrastructure-wide basis in a similar manner to the cross-cloud portability for Euclid defined here: https://github.com/astrodb/euclid-saas/blob/integration-wip/ansible/group_vars/cluster_cam/infra